blocks/subscriptions-block/components/PaymentIcons/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+export const PAYPAL = "PayPal";
+export const APPLEPAY = "ApplePay";
+export const GOOGLEPAY = "GooglePay";
+export const VISA = "visa";
+export const MASTERCARD = "mastercard";
+export const AMERICANEXPRESS = "amex";
+export const DISCOVER = "discover";
+export const DINERS = "diners";
+export const JCB = "jcb";
+export const UNIONPAY = "unionpay";
+
+// brands supported by StripeIntents https://docs.stripe.com/testing
+
+const PaymentIcon = ({ type }) => {
+	switch (type) {
+		case PAYPAL:
+			return <img src={require("./Icons/paypal-icon.svg")} alt="Buy now with PayPal" height={20} />;
+		case APPLEPAY:
+			return (
+				<img src={require("./Icons/applePay-icon.svg")} alt="Buy now with Apple Pay" height={19} />
+			);
+		case GOOGLEPAY:
+			return (
+				<img
+					src={require("./Icons/googlePay-icon.svg")}
+					alt="Buy now with Google Pay"
+					height={19}
+				/>
+			);
+		case VISA: {
+			return <img src={require("./Icons/visa-icon.svg")} alt={VISA} height={16} />;
+		}
+		case MASTERCARD: {
+			return <img src={require("./Icons/mastercard-icon.svg")} alt={MASTERCARD} height={16} />;
+		}
+		case AMERICANEXPRESS: {
+			return <img src={require("./Icons/amex-icon.svg")} alt={AMERICANEXPRESS} height={16} />;
```

Comment:
Has this been tested extensively? I don't think including svg's in this manner works. Those icons should be React components since Fusion Engine won't be able to serve static files from a block.

Reply:
I did a small test myself and got `<img src=[object Object] />` as the resulting output. Where exactly are the other instances of svg files being included directly? The only place I know of is the nav chain logo, but that's expecting a Photo center/3rd party url.